Index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Evaluations</title>
+
+    <!-- JS -->
+
+    <!-- Common -->
+
+    <script src="src\views\common\common.js"></script>
+    <script src="less.js" type="text/javascript"></script>
+
+    <!-- Events -->
+
+    <script src="src/events/login.js" type="text/javascript"></script>
+    <script src="src/events/evaluation.js" type="text/javascript"></script>
+    <script src="src/events/new-evaluation.js" type="text/javascript"></script>
+
+    <!-- Views -->
+
+    <script src="src/data/data.js" type="text/javascript"></script>
+    <script src="src/views/login/login.js" type="text/javascript"></script>
+    <script src="src/views/evaluation/evaluations.js" type="text/javascript"></script>
+    <script src="src/views/new-evaluation/new-evaluation.js" type="text/javascript"></script>
+    <script src="index.js" type="text/javascript"></script>
+    
+    
+    <!-- CSS -->
```

Comment:
you can import just one less file and use the @import instruction to load other css/less files inside